dsm_arnitel2045_175.xml.fdm_material

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Generic ABS profile. The data in this file may not be correct for your specific machine.
+-->
+<fdmmaterial xmlns="http://www.ultimaker.com/material">
+    <metadata>
+        <name>
+            <brand>DSM</brand>
+            <material>Elastomer</material>
```

Comment:
Elastomer is quite a generic description. From [this page](https://www.nexeo3d.com/en/arnitel-id-2045-grey.html) I understand that it's TPC (thermoplastic co-polymer). Perhaps that is more specific? Elastomer is a class of materials that includes all elastic polymers, including TPU and TPC.